### ISSUE TITLE
Tissue seems to fail on Python 3

### ISSUE THREAD

**Issue by lsmag** [wolph/tissue#11]
Hi,

I tried to use tissue in a project running on Python 3 and I got this error running the suite:

```
Traceback (most recent call last):
  File "manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/commands/test.py", line 30, in run_from_argv
    super(Command, self).run_from_argv(argv)
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/base.py", line 378, in run_from_argv
    parser = self.create_parser(argv[0], argv[1])
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/base.py", line 351, in create_parser
    self.add_arguments(parser)
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django/core/management/commands/test.py", line 62, in add_arguments
    test_runner_class.add_arguments(parser)
  File "/home/myuser/.virtualenvs/myproject/lib/python3.4/site-packages/django_nose/runner.py", line 260, in add_arguments
    parser.add_argument(opt_long, **option_attrs)
  File "/home/myuser/.pyenv/versions/3.4.3/lib/python3.4/argparse.py", line 1335, in add_argument
    action = action_class(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'metavar'
```

Could it be because tissue assumes optparse is being used? Python 2.7 still includes it, although it's deprecated. Anyway, I'll try to fix this and send a Pull Request later, is that ok for you?

**Comment by lsmag** [comment]
Actually, I noticed `0.3` is available on PyPI but here it's now on `0.9.2`. I reinstalled from this git repo and the issue continues to exist.

**Comment by wolph** [comment]
0.9.2 is also available on pypi actually: https://pypi.python.org/pypi/tissue

As for Python 3 support, it's working just fine for me so I have the feeling you have a different problem. It should be noted that argparse is included in Python since 2.7 so there's no need for it.

Looking at the traceback I think you probably need to report this error to django-nose instead :)

**Comment by wolph** [comment]
Closing this as I really can't reproduce it. If you're still having problems please let me know and I'll reopen it for you :)
